apps/theming-designer/src/components/app.scss

```diff
@@ -0,0 +1,38 @@
+@import '~office-ui-fabric-react/dist/sass/References';
+
+body,
```

Comment:
Can you use one of the css-in-js approaches instead? Sass files create a lot of headaches for build and we'd ideally like to get rid of them. 

For what you're doing, `mergeStyleSets` should work well. Here's an example to help you get started (note the special syntax for applying global styles to the `html` element):
```ts
const styles = mergeStyleSets({
  app: {
    display: 'flex',
    selectors: {
      ':global(html)': {
        padding: 0,
        margin: 0
      }
    }
  },
  sidebar: {
    flex: '0 0 400px',
    background: '#fff'
  }
});
```